### PR TITLE
New channel switches for Rec, Push, Buzzer, Email, FTP

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

New channel switches for Rec, Push, Buzzer, Email, FTP, see PR https://github.com/home-assistant/core/pull/91006

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/91006
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
For NVRs, a global switch for Record, Push, Buzzer, Email, and FTP will be available under the NVR device as well as a switch per channel of the NVR under the camera device. The respective feature will only be active for a given channel if both the global and that channel switch are enabled (as is also the case in the Reolink app/client).
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by starkillerOG** [comment]
@frenck I resolved the merge conflict, but that dismissed your approval.
Can you approve again?

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍 

../Frenck

**Pull request merged**
